src/auto-evo/AutoEvoRun.cs

```diff
@@ -174,6 +178,44 @@ public void Start()
         started = true;
     }
 
+    public void OneStep()
+    {
+        if (Running)
+            return;
+
+        started = true;
+
+        var timer = new Stopwatch();
+        timer.Start();
+
+        Running = true;
+
+        try
+        {
+            if (Step())
+                Finished = true;
+        }
+        catch (Exception e)
+        {
+            Aborted = true;
+            GD.PrintErr("Auto-evo failed with an exception: ", e);
+        }
+
```

Comment:
missing `timer.Stop` here?

Reply:
I see I did the same thing elsewhere, so you are free to ignore my comment here.